src/serializer/ResidualHeapVisitor.js

```diff
@@ -84,11 +84,13 @@ export class ResidualHeapVisitor {
   }
 
   visitObjectProperties(obj: ObjectValue): void {
-    /*
-    for (let symbol of obj.symbols.keys()) {
-      // TODO #22: visit symbols
+    for (let [symbol, propertyBinding] of obj.symbols) {
```

Comment:
there's a comment below "visit properties", so for consistency, say "visit symbols" as a comment here